pkg/segment/writer/logpacker.go

```diff
@@ -113,7 +119,6 @@ func parseNonJaegerRawJsonArray(currKey string, data []byte, tsKey *string,
 				return
 			}
 		case jp.String:
-
 			valUnescaped, encErr := jp.Unescape(value, jsParsingStackbuf[:])
```

Comment:
Do you need a similar fix here like you added in `parseRawJsonObject()`?

Reply:
addressed in the latest commit.